leapp/libraries/stdlib/__init__.py

```diff
@@ -32,3 +67,51 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd_info, buffer):
+    (_unused, fd_type) = fd_info
+    if os.getenv('LEAPP_DEBUG', '0') == '1':
+        if fd_type == call.STDOUT:
+            sys.stdout.write(buffer)
+        else:
+            sys.stderr.write(buffer)
+
+
+def new_call(args):
+    """
+    Call the Leapp's stdlib _call
+
+    :param args: Command to execute
+    :return: stdout output, 'utf-8' decoded
+    """
+    _id = str(uuid.uuid4())
+    result_data = None
+    try:
+        create_audit_entry('process-start', {'id': _id, 'parameters': args})
+        result_data = call._call(args, callback_raw=_logging_handler)
+    finally:
+        create_audit_entry('process-end', _id)
+        create_audit_entry('process-result', {'id': _id, 'parameters': args, 'result': result_data})
+    return result_data
+
+
+def new_checked_call(args, split=False):
+    """
+    Call the audited call and check result, split the result into multiple lines if requested
+
+    :param args: Command to execute
+    :param split: Split the output on newlines
+    :return: stdout output, 'utf-8' decoded, split by lines if split=True
+    """
+    result = new_call(args)
+    if result['exit_code'] != 0:
+        raise CalledProcessError(
+            message="A Leapp Command Error occurred. ",
+            command=str(args[0]),
```

Comment:
id' expect the whole 'args' here, not just the first command.

Reply:
sure, updated